src/ankh/cmd/ankh/main.go

```diff
@@ -262,3 +275,46 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		_, err := os.Stat(path)
+		if os.IsNotExist(err) {
+			log.Fatalf("Script not found: %s", path)
+		}
+		// check that the script is executable
+		err = syscall.Access(path, unix.X_OK)
+		if err != nil {
+			log.Fatalf("Permission denied: %s", path)
```

Comment:
this error should more descriptive, something like "Script is not executable:"

Reply:
Yeah I was just mirroring the err from syscall.Access
but you're right, that is slightly more informative, will change